### PR TITLE
Team 6 Blog

### PR DESCRIPTION
**Pull request by garified**

### CODE REVIEW

File: _contents/S0-L06.md [hunk 1]
```diff
@@ -1,44 +1,492 @@
...
+
+- https://arxiv.org/abs/2101.00027
+- Recent work has demonstrated that increased training dataset diversity improves general cross-domain knowledge and downstream generalization capability for large-scale language models. With this in mind, we present \textit{the Pile}: an 825 GiB English text corpus targeted at training large-scale language models. The Pile is constructed from 22 diverse high-quality subsets -- both existing and newly constructed -- many of which derive from academic or professional sources. Our evaluation of the untuned performance of GPT-2 and GPT-3 on the Pile shows that these models struggle on many of its components, such as academic writing. Conversely, models trained on the Pile improve significantly over both Raw CC and CC-100 on all components of the Pile, while improving performance on downstream evaluations. Through an in-depth exploratory analysis, we document potentially concerning aspects of the data for prospective users. We make publicly available the code used in its construction.
+
+# Section 1: The Pile
+
+In this section, we are going to talk introduce The pile, an open source dataset for diverse text for language modeling.
```
Comment (zhexjtu) [inline _contents/S0-L06.md:46]: In this section, we are going to introduce a paper: The ....

File: _contents/S0-L06.md [hunk 2]
```diff
@@ -1,44 +1,492 @@
...
+
+### More about the Pile
+
+Another goal of this work is to address ethical and bias concerns in AI research, while also promoting and standardizing the practice of engaging with AI ethics. The paper's analysis delves into various perspectives, including topic distribution, inappropriate content, sensitive content (gender, religion, race), and data authority. Readers interested in these aspects can explore the paper to find topics of interest.
+
+### Conclusion
+
+In conclusion, this work introduces a new open-source dataset that has been widely adopted in the research community since its release. The study demonstrates the dataset's capability enhancement by incorporating diverse categories of data through the evaluation process. Moreover, the work endeavors to address ethical and bias concerns in AI research, reflecting a commitment to responsible AI development.
+
+**Section 2 Mistral 7B**
```
Comment (zhexjtu) [inline _contents/S0-L06.md:112]: Please unify the caption font with other team members.

File: _contents/S0-L06.md [hunk 3]
```diff
@@ -1,44 +1,492 @@
...
+
+**Result:**
+
+Here is Mistral 7B performance on different tasks (comparing to other open source LLM)
+
+<img src="{{ site.baseurl }}/Lectures/S0-L06/images/mistral7B/Picture3.png" width="100%" height="100%">
+
+<img src="{{ site.baseurl }}/Lectures/S0-L06/images/mistral7B/Picture4.png" width="100%" height="100%">
+
+Mistral 7B performs equivalently to a Llama 2 that would be more than 3x its size. This is as much saved in memory and gained in throughput.
```
Comment (zhexjtu) [inline _contents/S0-L06.md:155]: equivalently to Llama2...

**Comment by qiyanjun** [comment]
are all issues solved?

**Comment by tempusaro** [comment]
> are all issues solved?

Yes, I think the phrasing issues mentioned in these three comments are fixed.

**Comment by tempusaro** [comment]
@zhexjtu Hello Zhe, I believe the phrasing issues mentioned in the three comments have been fixed. Could you please review it once more to confirm it's ready for merging this pull request?

**Comment by garified** [comment]
Yes, we have updated the file according to the feedback from Zhe.

**Pull request merged**